### PR TITLE
Refactor ProjectView 

### PR DESCRIPTION
**Pull request by kenjinp**
implements https://github.com/BlueBrain/nexus/issues/777

Note:
- uses resource List API, therefore the "visit elastic search query from this list" button has been removed.

### CODE REVIEW

File: src/shared/components/DropdownFilter/DropdownItem.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+const Option = AutoComplete.Option;
+
+export const TypeDropdownItem: React.FunctionComponent<{
+  count: number;
+  key: string;
+  label: string;
+}> = ({ count, key, label }) => {
+  return (
+    // Option does in fact have a label property, the ts warning is a lie.
+    // @ts-ignore (overloaded Options props with label, it works!)
```
Comment (julienmachon) [inline src/shared/components/DropdownFilter/DropdownItem.tsx]: Are you sure that you don't need to restart you ts-server or something. I'm pretty sure typescript never lies.

Comment (julienmachon) [reply src/shared/components/DropdownFilter/DropdownItem.tsx]: Looks like typescript is right after all

Screenshot from 2019-11-01 15-35-06

File: src/shared/components/DropdownFilter/DropdownItem.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    <Option key={key} value={key} label={label}>
```
Comment (julienmachon) [inline src/shared/components/DropdownFilter/DropdownItem.tsx]: So maybe ?
```suggestion
    <Option key={key} value={key} title={label}>
```

File: src/shared/store/actions/queryResource.ts [hunk 3]
```diff
@@ -94,88 +81,88 @@ const queryResourcesFailedAction: ActionCreator<FailedQueryAction> = (
...
-          schemas,
-          types
-        )
-      );
-    } catch (e) {
-      return dispatch(queryResourcesFailedAction(filterIndex, filterKey, e));
-    }
-  };
-};
+// export const queryResources: ActionCreator<ThunkAction> = (
```
Comment (julienmachon) [inline src/shared/store/actions/queryResource.ts]: We can assume that this can go then? :laughing:

Comment (kenjinp) [reply src/shared/store/actions/queryResource.ts]: yes!

File: src/shared/containers/FileUploadContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+      projectLabel,
+      {
+        '@id': file.name,
+        file: formData,
+        storage: storageId,
+      },
+      // TODO: fix! https://github.com/BlueBrain/nexus/issues/784
+      // @ts-ignore
+      {
+        extraHeaders: {
```
Comment (julienmachon) [inline src/shared/containers/FileUploadContainer.tsx]: can you remove this? we can fix it in the SDK. But his extra header isn't fixing anything. Also, not sure why there is a @ts-ignore as extraHeader is a valid option. The issue is created so we can get rid of the comment as well

Comment (kenjinp) [reply src/shared/containers/FileUploadContainer.tsx]: you got it!

**Comment by julienmachon** [comment]
Good job buddy! :100: / :100:

**Pull request merged**